docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+Static webpages are extremely easy to host, robust against traffic spikes and take-down attempts, and can often even be hosted for free on a number of services online.
+This helped make torrent indexers like ThePirateBay _decentralized_. In 2012, in an effort to ensure the longevity of the content on ThePirateBay and thanks to their recent conversion to magnet links, the site was replicated by users and [reduced to a 90 MB torrent file](https://torrentfreak.com/the-pirate-bay-returns-121213/) that anyone could copy and re-host.
+
+A magnet link works similar to how if you have ever clicked on an email address in the browser before and had it open up your default email client.
+
+If you have never seen an email link before, it looks like this:
+
+```html
+<a href="mailto:x@y.com">x@y.com</a>
+```
+
+If you click on that link, it will open up your default email client and create a new email to the address.
+
+Similarly, a magnet link looks like this:
+
+```html
+<a href="magnet:?xt=urn:btih:222FGW5I6KZUAEV4MM2WT4IT45YBPYBS&dn=ubuntu-23.04-live-server-amd64.iso&xl=2641264640&tr=https%3A%2F%2Ftorrent.ubuntu.com%2Fannounce">ubuntu-23.04-live-server-amd64.iso</a>
+```
+
+This link, when clicked, will launch your system default torrent client and begin downloading the file.
+
+### Torrent Clients
+
+If you were downloading a typical file from the internet, you would likely use a web browser to grab the file, save it to your computer and then end communication with the server.
+BitTorrent is different, as it is a peer-to-peer protocol, you are not just downloading a file from a server, you are also uploading the file to other users.
+To do so, you'll need a BitTorrent client, some software that can communicate with other users and share the file.
+
+There are many BitTorrent clients available, but the most popular these days are:
+  - [Transmission](https://transmissionbt.com/)
+  - [Deluge](https://deluge-torrent.org/)
+  - [qBittorrent](https://www.qbittorrent.org/)
+
+### Torrent Tracker
+
+In order to begin downloading a file via torrent, you need to connect with other "peers" or users in the swarm who have "pieces" of the file you want.
+A "[tracker](https://en.wikipedia.org/wiki/BitTorrent_tracker)" is a web server that keeps track of members in a swarm and aids in connecting users together based on which pieces of the file they have.
+
+When a torrent client initiates a download from a torrent file, it will first connect to the tracker and announce that it is joining the swarm and what pieces of the file it has available, if any.
+The tracker will respond with a list of peers that the client can connect to and begin downloading the file from. Once the client and a peer have connected, they no longer require the tracker to communicate with each other. However, the client will continue to communicate with the tracker to update it on the pieces it has available and to receive a list of new peers to connect with.
```

Comment:
[Proofreading]
```suggestion
When a torrent client initiates a download from a torrent file, it will first connect to the tracker. Next, the torrent cilent announces that it is joining the swarm and what pieces of the file it has available.
The tracker will then respond with a list of peers that the client can connect to and begin downloading the file from. Once the client and a peer have connected, they no longer require the tracker to communicate with each other. 
**Note:** The client will continue to communicate with the tracker to update it on the pieces it has available and to receive a list of new peers to connect with.
```